packages/react/src/components/DataTable/TableExpandRow.tsx

```diff
@@ -75,9 +75,15 @@ const TableExpandRow = React.forwardRef(
 
     // We need to put the slug before the expansion arrow and all other table cells after the arrow.
     let rowHasSlug;
```

Comment:
since this is internal it might be nice to update to use AILabel instead to keep consistency and shouldn't break anything on the adopter side
```suggestion
    let rowHasAILabel;
```